src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -113,6 +113,16 @@ public override bool Equals(object obj)
                    && Math.Abs(this.FarPlaneDistance - other.FarPlaneDistance) < 0.0001;
         }
 
+        internal bool isNaN()
```

Comment:
maybe `containsNaN` is clearer in this case because we are checking if any of the fields `isNan`

Reply:
👍